open_humans/models.py

```diff
@@ -26,21 +26,21 @@ def get_member_profile_image_upload_path(instance, filename):
     """
     Construct the upload path for a given member and filename.
     """
-    return 'member/%s/profile-images/%s' % (instance.user.id, filename)
+    return str('member/{0}/profile-images/{1}').format(instance.user.id, filename)
 
 def get_grant_project_image_upload_path(instance, filename):
     """
     Construct the upload path for an image for a ProjectGrant object.
     """
-    return 'grant-projects/%s/%s' % (instance.name, filename)
+    return str('grant-projects/{0}/{1}').format(instance.name, filename)
 
 def random_member_id():
     """
     Return a zero-padded string from 00000000 to 99999999 that's not in use by
     any Member.
     """
     def random_id():
-        return '%08d' % random.randint(0, 99999999)
+        return str('{0:d}').format(random.randint(0, 99999999))
```

Comment:
Are you sure this produces the right kind of padding? 

When running in an ipython shell I ended up with: 

```
In [52]: str('{0:d}').format(random.randint(0,99999999))
Out[52]: '91987'
```

Reply:
Ah, oops